test/features/app/spec.js

```diff
@@ -277,18 +285,27 @@ describe('Swagger spec', function() {
     });
 
     describe('routes', () => {
-
-        constructTests(spec.paths, defParams).forEach((testCase) => {
-            it(testCase.title, () => {
-                return preq(testCase.request)
-                .then((res) => {
-                    validateTestResponse(testCase, res);
-                }, (err) => {
-                    validateTestResponse(testCase, err);
-                });
-            });
+        beforeEach( () => {
+            server = new Server();
+            server.start()
+        })
+    
+        afterEach( () => server.stop() )
+    
+        initSpecAndParams()
```

Comment:
I think how it was was ok - we could reuse the spec received from a previous test. Test-suite-level isolation is good enough for us.

Reply:
See last comment. Due to the test creation function that didn't work.

Reply:
I renamed the function to make clear what it is intended to do but it's still there.